app/src/main/java/com/example/hcm_102_0006/android_project_m/view/adapter/AdapterShowCompany.java

```diff
@@ -0,0 +1,74 @@
+package com.example.hcm_102_0006.android_project_m.view.adapter;
+
+import android.content.Context;
+import android.databinding.DataBindingUtil;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.ViewGroup;
+
+import com.example.hcm_102_0006.android_project_m.R;
+import com.example.hcm_102_0006.android_project_m.data.model.MovieDetail;
+import com.example.hcm_102_0006.android_project_m.databinding.ItemCompanyBinding;
+import com.example.hcm_102_0006.android_project_m.ui.moviedetail.MovieDetailViewModel;
+
+import java.util.List;
+
+/**
+ * Created by hcm-102-0006 on 22/11/2017.
+ */
+
+public class AdapterShowCompany extends RecyclerView.Adapter<AdapterShowCompany.CompanyViewHolder> {
+    private List<MovieDetail.Company> mCompanies;
+    private Context mContext;
+    public static final String BUNDLE_COMPANY = "BUNDLE_COMPANY";
```

Comment:
2.8 Class member ordering

This is not the only correct solution, but it is recommended to use the following order :

Constants
Fields
Constructors
Override methods and callbacks (public or private)
Public methods
Private methods
Inner classes or interfaces